src/server/routes/fileProcessing.js

```diff
@@ -61,4 +62,31 @@ router.post('/:meter_id', upload.single('csvFile'), async (req, res) => {
 	}
 });
 
+router.post('/meters', async (req, res) => {
+	const validBody = {
+		type: 'object',
+		maxProperties: 1,
+		required: ['meters'],
+		properties: {
+			meters: {
+				type: 'array',
+				uniqueItems: false,
+				items: {
+					type: 'string'
+				}
+			}
+		}
+	};
+	if (!validate(req.body, validBody).valid) {
+		res.sendStatus(400);
+	} else {
+		try {
+			await insertMeters(req.body.meters.map(ip => ({ip})));
+			res.status(200).json({success: true});
+		} catch (err) {
+			res.status(403).json({success: false, message: 'Failed to upload meters.'});
```

Comment:
There is no reason to send the custom message. We are currently only sending a custom message if the client depends on a custom message to show a custom error notification.